common/djangoapps/edraak_i18n/middleware.py

```diff
@@ -3,7 +3,8 @@
 """
 
 from django.conf import settings
-from edraak_i18n.helpers import is_api_request
+from django.utils.translation import LANGUAGE_SESSION_KEY
+from .helpers import is_api_request
```

Comment:
Use full imports only.

```suggestion
from edraak_i18n.helpers import is_api_request
```

Reply:
Done